### ISSUE TITLE
Do not hide the Menu Bar when entering fullscreen (F11) on Windows

### ISSUE THREAD

**Issue by RedBlackAka** [microsoft/vscode#232780]
Do not hide the Menu Bar (that includes File, Edit, etc.) when entering fullscreen on Windows. It makes the fullscreen mode quite less usable. I really do like using it through!
![Image](https://github.com/user-attachments/assets/29c4db7a-5c0d-428c-a800-affc0b341b81)
![Image](https://github.com/user-attachments/assets/22c911a2-9b6f-4429-b3ca-cd8a1a60394e)

**Comment by RedBlackAka** [comment]
Realized that one can toggle it to always be visible in settings. I would however suggest making visible the default. And also keep the VSC icon on there. Thanks!

**Comment by benibenj** [comment]
I don't think we will be changing the default value here, however, we could add a context menu action on the title bar when in full screen to show/hide the menu

**Comment by benibenj** [comment]
I want to add a toggle Menu Bar action to the title bar when in full screen. Enabling sets it to `visible`, however, disabling is unclear. I was thinking to just set it to `classic`, but that could be confusing  to some users as toggling out of full screen would show the menu bar again. 

Maybe we need to call it `Menu Bar in Full Screen` which is a bit long and not nice, or have an action `Hide Menu Bar in Full Screen` (similar to the `Hide Custom Title Bar In Full Screen`)  but I would prefer it to stay a toggle as that's what it is when not in full screen.

@bpasero thoughts?

**Comment by bpasero** [comment]
I like to re-use existing settings and not have yet another one, even if the inverse is a bit confusing.

**Comment by sbatten** [comment]
@benibenj @bpasero If we are showing the title bar for some other reason in full screen (e.g. the command center or layout controls), I feel like hiding the menu bar in full screen with classic is actually strange. Essentially, that behavior originates from the native title bar. It would be nice to converge. If I were rewriting the settings, I'd probably have menubar be visible, toggle, compact. And have a new full screen titlebar visibility set to true / false.
